src/routes/viewer/Scene.svelte

```diff
@@ -49,5 +54,5 @@
 {/if}
 
 {#each $modelList as model}
-	<ModelView bind:url={model.url} bind:visible={model.visible} bind:color={model.color}/>
+	<ModelView bind:url={model.url} bind:visible={model.visible} bind:color={model.color} bind:bbox={model.bbox} bind:position={model.position} bind:scale={model.scale}/>
```

Comment:
this looks right, but I don't see where it would get adjusted across all of the objects.  Should there be a `Scene`-level bbox?

Maybe it would work if the bbox for each of the models was instantiated at load time, and then when `modelList` changed it changed a bbox that lived in the `Scene`?  Something like:

```
$: sceneBbox = $modelList.reduce( (accumulatedBbox, currentModel) => accumulatedBbox.Union(currentModel.bbox), new THREE.Bbox3D() );
```